### DISCUSSION TITLE
isRtlSupport naming

### DISCUSSION THREAD

**Discussion by Canato** [skydoves/Balloon#144] [General]
Just wanna make a small suggestion related to name of this amazing function.

I was using `isRtlSupport(true)` to set I want the support. 
But instead I should be passing if RTL is active now, more like `isRtlSupport(checkRTL())` 

Sadly this took me one day to realise =/ 

Not a big issue, but maybe have room for improvement? 

e.g.
`/** sets a flag for enabling rtl support */` ->  `/** sets a flag when rtl is enable/active */`
`isRtlSupport()` -> `isRtlActive()` 

🤔 what you think?

**Comment by skydoves** [comment]
@Canato 
Thanks for your suggestion :)
That means, integrating the functions of the supporting and checking for RTL would be better to use?

**Reply by Canato** [reply]
Different apps can have different approaches to allow RTL, maybe some don't use the OS RTL. For that is nice that we can setup directly.

And is good that the library keep the same approach, so maybe is just more a name changing 🤔
